package.json

```diff
@@ -32,19 +33,20 @@
     "@reach/auto-id": "^0.16.0",
     "@saleor/macaw-ui": "^0.8.0-pre.1",
     "@saleor/sdk": "^0.4.4",
-    "@floating-ui/react-dom-interactions": "^0.5.0",
     "@sentry/react": "^6.0.0",
     "@types/faker": "^5.1.6",
     "@uiw/react-color-hue": "0.0.34",
     "@uiw/react-color-material": "^0.1.0",
     "@uiw/react-color-saturation": "0.0.34",
     "apollo-upload-client": "^17.0.0",
+    "ci-info": "^3.7.1",
```

Comment:
issue: we already have `is-ci` please use that instead

Reply:
I don't think this is a great idea:
1. `is-ci` isn't used anywhere, so we don't need to limit ourselves ;)
2. `is-ci` actually uses... [`ci-info` under the hood](https://github.com/watson/is-ci/blob/master/index.js) and an old version which wasn't updated in 2 years 🙈